course_catalog/api.py

```diff
@@ -238,7 +243,6 @@ def digest_ocw_next_course(course_json, last_modified, uid, course_prefix):
                 uid,
                 run_serializer.errors,
             )
-            return
```

Comment:
I remove the return so that an invalid run breaks the transaction and prevents the course from being updated